docs/advanced_topics/performance.md

```diff
@@ -92,6 +92,80 @@ For some images, it may be beneficial to lazy load images, so the rest of the pa
 
 This optimisation is already handled for you for images in the admin site.
 
+## Template fragment caching
+
+Django supports [template fragment caching](https://docs.djangoproject.com/en/stable/topics/cache/#template-fragment-caching), which allows caching portions of a template. Using Django's `{% cache %}` tag natively with Wagtail can be [dangerous](https://github.com/wagtail/wagtail/issues/5074) as it can result in preview content being shown to end users. Instead, Wagtail provides 2 extra template tags which can be loaded from `wagtail_cache`:
```

Comment:
I've had to name the library `wagtail_cache` to avoid conflicting with [`wagtail-cache`](https://github.com/coderedcorp/wagtail-cache)'s `wagtailcache_tags`.